virttest/utils_net.py

```diff
@@ -3103,8 +3103,14 @@ def __arping(ip, macs, dev, timeout):
         mac_regex = "|".join("(%s)" % mac for mac in macs)
         regex = re.compile(r"\b%s\b.*\b(%s)\b" % (ip, mac_regex), re.I)
         arping_bin = utils_path.find_command("arping")
-        arping_cmd = "%s -f -c3 -w%d -I %s %s" % (arping_bin, int(timeout),
-                                                  dev, ip)
+        cmd = "%s --help" % arping_bin
+        if "-C count" in process.run(cmd, shell=True, ignore_status=True).stdout:
```

Comment:
Please append `verbose=False` and it is better to use `process.system_output` instead.

Reply:
Done.